src/pages/home/report/ReportActionCompose.js

```diff
@@ -474,7 +507,42 @@ class ReportActionCompose extends React.Component {
                                 <AttachmentPicker>
                                     {({openPicker}) => (
                                         <>
-                                            <View style={[styles.justifyContentEnd]}>
+                                            <View style={[
+                                                styles.dFlex, styles.flexColumn,
+                                                (this.state.isFullComposerAvailable || this.props.isComposerFullSize) ? styles.justifyContentBetween : styles.justifyContentEnd,
+                                            ]}
+                                            >
+                                                {this.props.isComposerFullSize && (
```

Comment:
This should be a ternary instead of `this.props.isComposerFullSize` and `!this.props.isComposerFullSize`. e.g.

this.props.isComposerFullSize ? (<components>) : (<otherComponents)

Reply:
Hmm, but what about this condition `(!this.props.isComposerFullSize && this.state.isFullComposerAvailable)`. With a ternary I would still need to check `this.state.isFullComposerAvailable`, so I kind of think this version is more readable.